src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tfoot = document.querySelector('tfoot');
+const columnIndex = 1;
```

Comment:
Ensure that the `columnIndex` variable is set to a valid index that exists in all rows of the table. If any row has fewer columns than `columnIndex + 1`, this will cause an error.